asyncio_mqtt/client.py

```diff
@@ -550,4 +563,4 @@ def _set_client_socket_defaults(client_socket: Optional[_PahoSocket]) -> None:
         return
     # At this point, we know that we got an actual socket. We change
     # some of the default options.
-    client_socket.setsockopt(socket.SOL_SOCKET, socket.SO_SNDBUF, 2048)
+    client_socket.setsockopt(*socket_param)
```

Comment:
Lastly, we apply all the socket options:
```
for socket_option in socket_options:
    client_socket.setsockopt(*socket_option)
```

This way, the user can define multiple socket options. :+1: